### DISCUSSION TITLE
Rooms to Google Home

### DISCUSSION THREAD

**Discussion by Thimboo** [t0bst4r/home-assistant-matter-hub#294] [Q&A] [answered]
Hi everyone,
If I see it correctly, the rooms are not transferred to Google Home.
Is this the case, or have I overlooked something?

**Comment by t0bst4r** [comment]
nope it's not. I have never thought about that, and i dont know (yet) if that's (currently) possible with matter.

**Reply by Apollon77** [reply, marked as answer]
it currently is not possible from my knowledge. There are theoretical options using the *Label clusters but especially these are not used by anyone. In fact the concept of Matter is that a Device do not have such knowledge and the controllers "own" these things like Rooms and final names,. The Device is "just a device" - also with a Bridge.
